infrastructure/templates/template.json

```diff
@@ -376,6 +376,13 @@
         "description": "Gov UK Notify service template ID for approver notifications"
       }
     },
+    "immediatePublicationOfScheduledReleasesEnabled": {
+      "type": "string",
```

Comment:
Should this be a boolean type instead of `string`?

Reply:
Changed to boolean thanks